spec/CoopTilleuls/Bundle/YmlpBundle/Ymlp/YmlpClientSpec.php

```diff
@@ -47,7 +49,7 @@ function let(
 
     function it_is_initializable()
     {
-        $this->shouldHaveType('CoopTilleuls\Bundle\YmlpBundle\Ymlp\YmlpClient');
+        $this->shouldHaveType(YmlpClient::class);
```

Comment:
Can you revert this please? This breaks the support of PHP 5.4. I will update to 5.5 in another PR with the support of Guzzle 6.